apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -486,6 +488,11 @@ class PageTemplateModal extends Component {
 										homepageTemplates,
 										__( 'Home Pages', 'full-site-editing' )
 									) }
+								{ comingSoonTemplates &&
+									this.renderTemplatesList(
+										comingSoonTemplates,
+										__( 'Coming Soon Pages', 'full-site-editing' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **69** times:
`translate( 'Coming Soon' )` *ES Score: 6*